### PR TITLE
Multiple ParseJson

### PR DESCRIPTION
**Pull request by marcenacp**
Add the capacity to filter multiple JSON. When reading a JSON, I also parse all needed fields.

### CODE REVIEW

**Comment by ccl-core** [comment]
Can it be that some of the commits in this PR got somehow merged with the commits in #165 ?

**Review by ccl-core — APPROVED** [review]
Thank you Pierre!

File: python/ml_croissant/ml_croissant/_src/structure_graph/nodes/__init__.py [hunk 1]
```diff
@@ -3,12 +3,16 @@
 from ml_croissant._src.structure_graph.nodes.file_set import FileSet
 from ml_croissant._src.structure_graph.nodes.metadata import Metadata
 from ml_croissant._src.structure_graph.nodes.record_set import RecordSet
-from ml_croissant._src.structure_graph.nodes.source import Source
-from ml_croissant._src.structure_graph.nodes.source import Transform
+from ml_croissant._src.structure_graph.nodes.source import (
```
[the remark this answers is not in the document]
Comment (marcenacp) [reply python/ml_croissant/ml_croissant/_src/structure_graph/nodes/__init__.py:6]: Acknowledged.

I have https://github.com/mlcommons/croissant/pull/166 to automatically format imports

**Comment by marcenacp** [comment]
> Can it be that some of the commits in this PR got somehow merged with the commits in #165 ?

Both PRs are dependant. You can review commit by commit by clicking the `Commits` tab, then selecting a commit.

**Comment by marcenacp** [comment]
Thank you for the review!

**Pull request merged**